Fixed NavBar

Alfredcrmn opened the pull request:
The NavBar is now fixed, so when the user scrolls down, it remains at the top of the viewport. 

Added a z-index of 1000 to ensure the NavBar stacks at the top of the page's elements.

Alfredcrmn committed:
Fixed NavBar

Added CSS for the NavBar to be fixed. It works both on PC and Mobile.

hanrodz reviewed (changes requested):
These changes look good but when navigating to the different sections the headers are cropped. The added spacing should be accounted for.

Not very familiar with the navigation bar implementation but I think the fix might be in or around `layouts/App.tsx`.

<table>
<tr>
<td>
<img alt="Screenshot of Partners section where the header is cropped" src="https://github.com/user-attachments/assets/2206069d-4bbf-47e5-a361-12b388e8bf68" />
</td>
<td>
<img alt="Screenshot of FAQ section where the header is cropped" src="https://github.com/user-attachments/assets/2f3048ec-026e-4de1-b343-d7b8d62f5aa5" />
</td>
</tr>
</table>

Alfredcrmn committed:
Fixed navbar

added scroll-padding-top. sync spacer height and z-index with global.css values

hanrodz reviewed (approved):
LGTM 👍

hanrodz commented:
@Alfredcrmn feel free to merge as soon as the required checks succeed 😸
